scripts/do.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 # This script is an interface to any of the methods of lib.sh
 # Call this script as "do.sh method_from_lib" to execute any function from that library
-set -eu
+set -e
```

Comment:
checking for unbound variables is a good thing, we should fix whatever is accessing unbound variables instead